packages/canary/src/components/text.tsx

```diff
@@ -33,7 +33,7 @@ const textVariants = cva('text-base', {
       both: ''
     },
     truncate: {
-      true: 'truncate text-ellipsis text-nowrap'
+      true: 'truncate'
```

Comment:
If we remove this style, I'm not sure the behavior will remain the same in other places where it's used. It might be better to leave it as is, or thoroughly check all instances where truncation is currently applied.

Reply:
this class simply already has the necessary values in it
![Google Chrome 2024-11-07 12 43 07](https://github.com/user-attachments/assets/17c81118-0fcd-451f-8321-b3a514f910cc)